.github/workflows/client-celo.yml

```
@@ -0,0 +1,79 @@
+name: Go / Celo
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

We can remove the filter and run iyt on all PRs.
```suggestion
  pull_request:
```

---

Please see: https://github.com/keep-network/keep-ecdsa/commit/03aafa389975e71405a6a3441ab777ea8c9ce46d